src/state/slices/assetsSlice/assetsSlice.ts

```diff
@@ -40,22 +40,26 @@ export const defaultAsset: Asset = {
 export type UpsertAssetsPayload = Omit<AssetsState, 'relatedAssetIndex'>
 
 export const assets = createSlice({
-  name: 'asset',
+  name: 'assets',
   initialState,
-  reducers: {
-    clear: () => initialState,
-    upsertAssets: (state, action: PayloadAction<UpsertAssetsPayload>) => {
+  selectors: {
+    selectAssetsById: state => state.byId,
+    selectAssetIds: state => state.ids,
```

Comment:
can add `selectRelatedAssetIndex` and replace `related-assets-selectors:L23`

Reply:
95438f969b